README.rst

```diff
@@ -8,6 +8,15 @@
 .. image:: https://codecov.io/gh/nipy/dipy/branch/master/graph/badge.svg
   :target: https://codecov.io/gh/nipy/dipy
 
+.. image:: https://img.shields.io/github/release/qubyte/rubidium.svg
+  :target: https://github.com/nipy/dipy/releases/latest
```

Comment:
Since it does not work, we can replace it by:

```
.. image:: https://badge.fury.io/py/dipy.svg
  :target: https://badge.fury.io/py/dipy
```

Reply:
It was not showing the right release because it was pointing to another repo (my bad!), but this is still not working:
https://img.shields.io/github/release/nipy/dipy.svg
API call:
https://api.github.com/repos/nipy/dipy/releases/latest
Compare the API call answer to
https://api.github.com/repos/insightsoftwareconsortium/itk/releases/latest

In any case, for now I ignore what's wrong, and I'm leaving the PyPI version (using the "official" or more common badge server Serge):
```
.. image:: https://img.shields.io/pypi/v/dipy.svg
  :target: https://pypi.python.org/pypi/dipy
```